WordPress/src/main/res/layout/my_site_fragment.xml

```diff
@@ -320,6 +320,65 @@
 
     </ScrollView>
 
+    <LinearLayout
+        android:id="@+id/no_site_view"
+        android:layout_width="match_parent"
```

Comment:
It would be good to add `android:visibility="gone"` here to avoid having it displayed during the initial layout pass (probably nothing the user would see, but it's a minor unnecessary performance hit).